aie_runtime_lib/AIE/exp_lut.cpp

```diff
@@ -214,3 +214,14 @@ alignas(aie::vector_decl_align) int16 softmax_flut_cd[512] = {
     16422, 16423, 16423, 16424, 16419, 16420, 16421, 16421, 16422, 16423, 16423,
     16424, 16425, 16425, 16426, 16427, 16427, 16428, 16429, 16429, 16425, 16425,
     16426, 16427, 16427, 16428, 16429, 16429};
+
+alignas(aie::vector_decl_align) unsigned char m_inv_lut[128] = {
```

Comment:
Now that this file contains lut tables for exp and inv, it should probably be renamed. How about lut_based_ops.cpp and lut_based_ops.h ?

Reply:
Also, can we strive towards some consistency in the naming of these global variables?